server/src/main/java/org/spine3/server/entity/Entity.java

```diff
@@ -223,6 +252,62 @@ public Timestamp whenModified() {
     }
 
     /**
+     * Ensures that the type of the {@code entityId} is supported.
+     *
+     * @param entityId the ID of the entity to check
+     * @throws IllegalArgumentException if the ID is not of one of the supported types
+     */
+    public static <I> void checkIdType(I entityId) {
+        final Class<?> idClass = entityId.getClass();
+        if (SUPPORTED_ID_TYPES.contains(idClass)) {
+            return;
+        }
+        if (!Message.class.isAssignableFrom(idClass)){
+            throw unsupportedIdType(idClass);
```

Comment:
It is strange that CodeCov shows this line as not tested, because a test for this case was added months ago.